content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -79,55 +79,54 @@ Recent key learnings:
 
 ## Strategy and Plans
 
-### FY22 Q4 goals
-
-These tie back to [FY22 Q4 Product/Engineering OKRs](../../../goals/2022_Q4.md#product-engineering)
+### Themes
```

Comment:
Are these scoped to FY23 Q1? @jyavorska, it might be worth calling that out in all our strategy pages so it's clear what the time frame is?

Reply:
It's scoped to the strategy pages themselves, which are scoped to the next six months or a year or so. I'll create a separate PR to add something to the header of each strategy page that indicates that.

Reply:
Sounds good, thanks!